qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/presenters/Presenter.groovy

```diff
@@ -18,7 +18,7 @@ import life.qbic.portal.portlet.offers.update.UpdateOfferOutput
  * @author: Jennifer Bödker
  *
  */
-class Presenter implements SearchOffersOutput, CreateOfferOutput, UpdateOfferOutput, CreateCustomerOutput, UpdateCustomerOutput {
```

Comment:
why did you remove the output? I would expect all use cases to have an output here.

Reply:
I moved it to its own presenter ;)